compass/landice/tests/ismip6_forcing/ocean_thermal/process_thermal_forcing.py

```diff
@@ -0,0 +1,308 @@
+
+        # interpolate and rename the ismip6 thermal forcing data
+        remapped_file_temp = "remapped.nc"  # temporary file name
+
+        # call the function that reads in, remap and rename the file.
+        print("Calling a remapping function...")
+        self.remap_ismip6thermalforcing_to_mali(input_file,
+                                                remapped_file_temp,
+                                                mali_mesh_name,
+                                                mali_mesh_file, method_remap)
+
+        # call the function that renames the ismip6 variables to MALI variables
+        print("Renaming the ismip6 variables to mali variable names...")
+        self.rename_ismip6thermalforcing_to_mali_vars(remapped_file_temp,
+                                                      output_file)
+
+        print("Remapping and renamping process done successfully. "
+              "Removing the temporary file 'remapped.nc'")
+
+        # remove the temporary combined file
+        os.remove(remapped_file_temp)
+
+        # place the output file in appropriate directory
+        if output_base_path == "NotAvailable":
+            return
+
+        if not os.path.exists(output_path):
+            print("Creating a new directory for the output data")
+            os.makedirs(output_path)
+
+        src = os.path.join(os.getcwd(), output_file)
+        dst = os.path.join(output_path, output_file)
+        shutil.copy(src, dst)
+
+    def remap_ismip6thermalforcing_to_mali(self, input_file,
+                                           output_file, mali_mesh_name,
+                                           mali_mesh_file, method_remap):
+        """
+        Remap the input ismip6 thermal forcing data onto mali mesh
+
+        Parameters
+        ----------
+        input_file: str
+            ismip6 thermal forcing data on its native polarstereo 8km grid
+        output_file : str
+            ismip6 data remapped on mali mesh
+        mali_mesh_name : str
+            name of the mali mesh used to name mapping files
+        mali_mesh_file : str, optional
+            The MALI mesh file if mapping file does not exist
+        method_remap : str, optional
+            Remapping method used in building a mapping file
+        """
+
+        # check if a mapfile
+        mapping_file = f"map_ismip6_8km_to_{mali_mesh_name}_{method_remap}.nc"
+
+        if not os.path.exists(mapping_file):
+            # build a mapping file if it doesn't already exist
+            build_mapping_file(self.config, self.cores, self.logger,
+                               input_file, mapping_file, mali_mesh_file,
+                               method_remap)
+        else:
+            print("Mapping file exists. Remapping the input data...")
+
+        # remap the input data
+        args = ["ncremap",
+                "-i", input_file,
+                "-o", output_file,
+                "-m", mapping_file]
+
+        subprocess.check_call(args)
```

Comment:
Again, this needs to use `mpas_tools.logging.check_call` and pass `self.logger`.  Otherwise, output goes to the terminal instead of the log file.